src/gridstack.ts

```diff
@@ -2181,7 +2181,7 @@ export class GridStack {
       if (node._temporaryRemoved) return; // handled by dropover
       let distance = ui.position.top - node._prevYPix;
       node._prevYPix = ui.position.top;
-      if (this.opts.draggable.scroll === false) {
+      if (this.opts.draggable.scroll) {

```

Comment:
!== false 
still better in case we fail to init that var (I know it's it in the defaul now). I'll update as I need to a release for work :)